spec/models/project_spec.rb

```diff
@@ -21,6 +21,30 @@
     expect(sub_project2.position).to eq 2
   end
 
+  describe "#archived?" do
+    it "returns true if archived" do
+      expect(Project.new(status: "archived").archived?).to eql(true)
```

Comment:
maybe we can do `expect(Project.new(status: "archived")).to be_archived`? I think it's cleaner to read and it will check that `archived?` returns `true` anyway, since by convention methods ending with `?` should return true/false I think it's ok to not be explicit on the matcher against true/false

also maybe the test name can match the other `it "returns true if status is 'archived'" do`, so on both cases we reference the status attribute